### PR TITLE
feat(Emotion): Add Emotion Dependency to Next WEB-982

### PR DESCRIPTION
**Pull request by codecaaron**
## Overview
- Adds emotions styled / core as base packages and adds them as peer dependencies for all probable consumers.
- Converts Badge as a simple example to show it works.

### PR Checklist

- [ ] Related to Abstract designs:
- [x] Related to JIRA ticket: WEB-982
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2119,7 +2131,7 @@
     "@emotion/serialize" "^0.11.15"
     "@emotion/utils" "0.11.3"
 
-"@emotion/styled@^10.0.17":
```
Comment (codecaaron) [inline yarn.lock:2122]: Storybook's version.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@1.7.51-alpha.db52f4.0<br/>@codecademy/gamut-styles@4.1.1-alpha.db52f4.0<br/>@codecademy/gamut@12.0.2-alpha.db52f4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f1a1e7f8a12db019c2d398b--codecademy-styleguide.netlify.app>https://5f1a1e7f8a12db019c2d398b--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5f1a1e7f8a12db019c2d398b>Deploy Logs</a></p>

File: packages/gamut/src/Badge/index.tsx [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
+
+export const Badge = styled.span`
+  background: ${colors.blue[500]};
+  border-radius: 3px;
+  color: ${colors.white};
+  display: inline-block;
+  font-size: 0.85em;
+  margin: 0 0.5em;
+  padding: 0.25em 0.75em;
+`;
```
Comment (JoshuaKGoldberg) [inline packages/gamut/src/Badge/index.tsx:12]: wowee this is so much cleaner...

_"Were it so easy [all the time]"_ - Thel 'Vadamee

**Pull request merged**